src/Workspaces/CoreTest/Microsoft.CodeAnalysis.Workspaces.UnitTests.csproj

```diff
@@ -27,6 +27,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="System.Buffers" Version="$(SystemBuffersVersion)" />
+    <PackageReference Include="Xunit.Combinatorial" Version="$(XunitCombinatorialVersion)" PrivateAssets="all" />
```

Comment:
```suggestion
    <PackageReference Include="Xunit.Combinatorial" Version="$(XunitCombinatorialVersion)" />
```

Reply:
What's our guideline on this? I ask only because I copied this from another unit test file and that's what it did...